Firestore/CHANGELOG.md

```diff
@@ -26,6 +26,9 @@
   is now the default.)
 - [feature] Added new `mergeFields:(NSArray<id>*)` override for `set()`
   which allows merging of a reduced subset of fields.
+- [fixed] Fixed an issue where `FirestoreSettings` would accept a concurrent
```

Comment:
It looks like we forgot to submit a revision of this file that promoted all the "Unreleased" things to a version with the [5.0.0 release of Firebase](https://firebase.google.com/support/release-notes/ios). Could you move everything above this entry into a `v0.12.0` section?

Reply:
Right but your new change did not make it into that release so it would still be in the unreleased section.